xAPI-Data.md

```diff
@@ -1571,7 +1571,7 @@ Used to record the time at which the experience described in the Statement.
 ###### Requirements
 
 * For requirements pertaining to the Timestamp data type, see [ISO 8601 Timestamps](#timestamps) below.
-* The stored property MUST be set by the LRS; An LRS MUST validate and then overwrite any value currently in the 
+* The stored property MUST be set by the LRS; An LRS SHOULD validate and then overwrite any value currently in the 
```

Comment:
I think there just needs to be a slight change here. Based on the call on Wednesday I understood this to be `An LRS SHOULD validate and then MUST overwrite any value currently in the stored property of a Statement it received`

The key part being the `MUST` overwrite, as right now it reads as a `SHOULD` for overwriting.